### DISCUSSION TITLE
'pipenv' is not recognized as the name of a cmdlet, function, script file, or operable program

### DISCUSSION THREAD

**Discussion by Siddhant963** [Atri-Labs/atrilabs-engine#634] [Help: Installation & Start] [answered]
PS C:\Users\Hp> cd my_atri_app
PS C:\Users\Hp\my_atri_app> pipenv install atri==0.0.78
pipenv : The term 'pipenv' is not recognized as the name of a
cmdlet, function, script file, or operable program. Check the
spelling of the name, or if a path was included, verify that the
path is correct and try again.
At line:1 char:1
+ pipenv install atri==0.0.78
+ ~~~~~~
    + CategoryInfo          : ObjectNotFound: (pipenv:String) []
   , CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException

**Comment by darshitac11** [comment, marked as answer]
It has been answered here: https://docs.atrilabs.com/troubleshoot/installation_error#the-term-pipenv-is-not-recognized-as-the-name-of-a-cmdlet-function-script-file-or-operable-program
